### DISCUSSION TITLE
Local Graph filters

### DISCUSSION THREAD

**Discussion by BahneGork** [oleeskild/obsidian-digital-garden#193] [Q&A]
hey, 

Is it possible to tweak the local graph?
for instance, include non-existant links?

**Comment by uroybd** [comment]
Hi, nice suggestion. Seems doable. I'll try to do so in my free time.

**Reply by BahneGork** [reply]
nice :D i tried a quick look around in settings but i couldnt identify where to go about it.

**Reply by uroybd** [reply]
There is none atm. I have to develop the feature first.
